src/main/java/com/ongvida/api/exceptionhandler/ApiExceptionHandler.java

```diff
@@ -0,0 +1,56 @@
+package com.ongvida.api.exceptionhandler;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.MessageSource;
+import org.springframework.context.i18n.LocaleContextHolder;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.FieldError;
+import org.springframework.validation.ObjectError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+import com.ongvida.domain.exception.BusinessException;
+@ControllerAdvice 
+public class ApiExceptionHandler extends ResponseEntityExceptionHandler {
+	@Autowired
+	private MessageSource messageSource;
+	
+	@ExceptionHandler(BusinessException.class)
+	public ResponseEntity<Object> handleNegocio(BusinessException ex, WebRequest request) {
+		var status = HttpStatus.BAD_REQUEST;
+		
+		var bug = new Bugs();
+		bug.setStatus(status.value()); 
+		bug.setTitulo(ex.getMessage());
+		bug.setDataHora(LocalDateTime.now());
+		return handleExceptionInternal(ex, bug, new HttpHeaders(), status, request);
+		
+	}
+	
+@Override
+	protected ResponseEntity<Object> handleMethodArgumentNotValid(MethodArgumentNotValidException ex,
+			HttpHeaders headers, HttpStatus status, WebRequest request){
+	var fields = new ArrayList<Bugs.Campo>();
+	for (ObjectError error : ex.getBindingResult().getAllErrors()) {
+		String name = ((FieldError) error).getField();
+		String message = messageSource.getMessage(error, LocaleContextHolder.getLocale());
+		fields.add(new Bugs.Campo(name,message));
+	}
```

Comment:
Esse nome Bugs não está bom, tá, porque na verdade não é pra isso que serve essa classe. Essa classe serve para você usar objetos dela para organizar nela os campos que "deram erro", isto é, você, por meio dela usa pega todos os campos que estão com erro ( note que você define um ArrayList ali em Bugs.Campo - na linha 40 e vai setar ele dentro de fields). Sugiro ao invés de colocar "Bug"  colocar "ErrorField". Tem mais sentido com a finalidade da classe

```suggestion
	var fields = new ArrayList<ErrorField.Campo>();
	for (ObjectError error : ex.getBindingResult().getAllErrors()) {
		String name = ((FieldError) error).getField();
		String message = messageSource.getMessage(error, LocaleContextHolder.getLocale());
		fields.add(new Bugs.Campo(name,message));
	}
```